src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassUploadDocService.java

```diff
@@ -37,8 +47,8 @@ public class CafcassUploadDocService {
 
     public static final List<String> ALLOWED_FILE_TYPES = List.of("pdf", "docx");
     public static final List<String> ALLOWED_TYPE_OF_DOCS = List.of("16_4_Report", "CIR_Part1", "CIR_Part2", "CIR_Review", "CMO_report",
-        "Contact_Centre_Recordings", "Correspondence", "Direct_work", "Enforcement_report", "Enquiry_from_Foreign_Court", "FAO_Report",
-        "FAO_Workplan", "High_Court_Team_Template", "Letter_from_Child", "Other_Non_Section_7_Report", "Parental_Order_Report", "Position_Statement",
+        "Contact_Centre_Recordings", "Correspondence", "Direct_work", "Enforcement_report", "FAO_Report",
```

Comment:
Why is this list changed?

Reply:
As per the requirement we have changed it